wire-tests-swift/no-manifest/src/main/swift/LargeMessage.swift

```diff
@@ -0,0 +1,456 @@
+        get {
+            storage.value6
+        }
+        set {
+            storage.value6 = newValue
+        }
+    }
+    public var value7: String? {
+        get {
+            storage.value7
+        }
+        set {
+            storage.value7 = newValue
+        }
+    }
+    public var value8: String? {
+        get {
+            storage.value8
+        }
+        set {
+            storage.value8 = newValue
+        }
+    }
+    public var value9: String? {
+        get {
+            storage.value9
+        }
+        set {
+            storage.value9 = newValue
+        }
+    }
+    public var value10: String? {
+        get {
+            storage.value10
+        }
+        set {
+            storage.value10 = newValue
+        }
+    }
+    public var value11: String? {
+        get {
+            storage.value11
+        }
+        set {
+            storage.value11 = newValue
+        }
+    }
+    public var value12: String? {
+        get {
+            storage.value12
+        }
+        set {
+            storage.value12 = newValue
+        }
+    }
+    public var value13: String? {
+        get {
+            storage.value13
+        }
+        set {
+            storage.value13 = newValue
+        }
+    }
+    public var value14: String? {
+        get {
+            storage.value14
+        }
+        set {
+            storage.value14 = newValue
+        }
+    }
+    public var value15: String? {
+        get {
+            storage.value15
+        }
+        set {
+            storage.value15 = newValue
+        }
+    }
+    public var value16: String? {
+        get {
+            storage.value16
+        }
+        set {
+            storage.value16 = newValue
+        }
+    }
+    /**
+     * A large message with a redacted description causes a Swift compiler error.
+     */
+    public var description: String? {
+        get {
+            storage.description
+        }
+        set {
+            storage.description = newValue
+        }
+    }
+    public var unknownFields: Foundation.Data {
+        get {
+            storage.unknownFields
+        }
+        set {
+            storage.unknownFields = newValue
+        }
+    }
+
+    public init(configure: (inout Self.Storage) -> Swift.Void = { _ in }) {
+        self.storage = Storage(
+                configure: configure
+                )
+    }
+
+}
+
+#if !WIRE_REMOVE_EQUATABLE
+extension LargeMessage : Equatable {
+}
+#endif
+
+#if !WIRE_REMOVE_HASHABLE
+extension LargeMessage : Hashable {
+}
+#endif
+
+extension LargeMessage : Sendable {
+}
+
+extension LargeMessage : ProtoDefaultedValue {
+
+    public static var defaultedValue: LargeMessage {
+        LargeMessage()
+    }
+}
+
+#if !WIRE_REMOVE_REDACTABLE
+extension LargeMessage : Redactable {
+
+    public enum RedactedKeys : String, RedactedKey {
+
+        case description
+
+    }
+
+    public var description_: String {
+        return storage.description
```

Comment:
hmmm is this correct and intended @lickel ? I think there was a different intention to add this property here?

Reply:
I think this is also actually wrong and perhaps not intended?

Reply:
OK the underlying invocation is correct however, renaming this to `description_` however will make invocations like `print(x)` to no longer print the message correct in the logs.

Reply:
@lickel we can discuss this further. As this PR is now this is not the correct fix.

I think we need to append `_` in the property we output for the field instead of renaming this.

Reply:
I would _unconditionally_ rename the proto property for `description`, similar to `Type`

Reply:
Type_ I think was meant to be if the _message_ is named as such. This is more about a field named `description`.

However, we agree 100% that we should be renaming the property we generate for the field instead of this because this would break logging of redatcted fields.